create-snowpack-app/cli/createSnowpackApp.js

```diff
@@ -174,11 +176,11 @@ const installedTemplate = isLocalTemplate
     function installProcess(packageManager) {
       switch (packageManager) {
         case 'npm':
-          return execa('npm', ['install', '--loglevel', 'error'], npmInstallOptions);
+          return execa('npm', ['install', '--loglevel', 'error', verboseArg], npmInstallOptions);
```

Comment:
Apologies for the delay looking at this, but can you confirm the output that you see when running this with the `--verbose` flag? We already pass `--silent` and `--loglevel error` to those same commands, so sending `--verbose` at the same time would be contradictory / error prone. 

Instead, can you decide between the two here:

```
return execa('npm', ['install', ...(verbose ? ['--verbose'] : ['--loglevel', 'error'])], {
```

Reply:
Yep. i guess for npm `--loglevel verbose` and `--verbose` does the same thing.

Updated PR. Thanks